### ISSUE TITLE
Assertion fails when using the built-in "re" module

### ISSUE THREAD

**Issue by lggwettmann** [davidfoerster/aptsources-cleanup#30]
I get this:
```
  File "aptsources-cleanup.pyz/aptsources_cleanup/util/gettext.py", line 213, in ChoiceHighlighters
    builtins.__dict__['_'] = self.gettext
AssertionError: "unprintable_delimiters" contains regex meta-characters.
```

**Comment by davidfoerster** [comment]
Ugh, looks like `re.escape` (before Python 3.9) [escapes all non-ASCII characters](https://bugs.python.org/issue29995). I’ll release a fix/work-around some time soon.

In the mean time you can skip assertion checks if you invoke Python with the `-O` flag. I also rebuilt the PYZ bundle for the [current release](/davidfoerster/aptsources-cleanup/releases/tag/v0.1.7.2) to include a shebang line, so you can actually execute it as described in [*Usage*](/davidfoerster/aptsources-cleanup#usage) which also skips assertion checks.

**Comment by davidfoerster** [comment]
Fixed in aac69ce02ede727327645862af7d09ce86d77b23
